modules/apps/powwow/powwow-portlet/docroot/meetings/view_occurrences.jsp

```diff
@@ -60,17 +62,57 @@ portletURL.setParameter("backURL", backURL);
 			<liferay-ui:message key="occurrences" />
 		</dt>
 		<dd>
-			Placeholder
+			<liferay-ui:search-container
+				total="<%= PowwowMeetingOccurrenceLocalServiceUtil.countByPowwowMeetingId(powwowMeetingId) %>"
```

Comment:
since we already find all `powwowMeetingOccurrences`, then it's not helpful to call count method here. let's use `List.size()`.